### DISCUSSION TITLE
stacking simulations

### DISCUSSION THREAD

**Discussion by logixworx** [dimforge/rapier#472] [Q&A]
I am evaluating rapier3d for use in networked simulations.

Is rapier3d able to stack simulations without issues? Ex: room server - each room has their own rapier3d simulation.

**Comment by sebcrozet** [comment]
Having multiple rapier simulation (with their own physics pipelines, rigid-body sets, etc.) will work without issues. Of course, it’s up to you to deal with objects transitioning from one physics simulation to the other.
